code/modules/atmospherics/machinery/components/gas_recipe_machines/atmos_machines_recipes.dm

```diff
@@ -74,17 +74,17 @@ GLOBAL_LIST_INIT(gas_recipe_meta, gas_recipes_list())
 	min_temp = 15
 	max_temp = 35
 	reaction_type = ENDOTHERMIC_REACTION
-	energy_release = 300000
-	requirements = list(/datum/gas/freon = 50, /datum/gas/plasma = 40, /datum/gas/oxygen = 30)
-	products = list(/obj/item/stack/sheet/hot_ice = 1)
+	energy_release = 3000000
+	requirements = list(/datum/gas/freon = 40, /datum/gas/plasma = 70, /datum/gas/oxygen = 30)
+	products = list(/obj/item/stack/sheet/hot_ice = 2)
```

Comment:
How's the plasma -> plasma ratio here? since hot ice can be heated into plasma iirc

Reply:
ah yes, is currently 150, i'll raise the plasma requirement for it

Reply:
1 Sheet of hot ice is 25 units of hot ice slush. 1 unit of hot ice slush is 12 units of plasma,  which directly convert to the gas.
So two sheets of hot ice are 25 * 12 moles of plasma, or 300.

I'd be alright with lowering the amount of plasma in hot ice, cause god damn that's a lot, but I'd like this reaction to at least be balanced.